dhcpv4/options.go

```diff
@@ -344,6 +344,10 @@ func getOption(code OptionCode, data []byte, vendorDecoder OptionDecoder) fmt.St
 		var u Uint16
 		d = &u
 
+	case OptionAutoConfigure:
```

Comment:
NIT: maybe sort by option code here

Reply:
Moved OptionAutoConfigure to the right place (but not all the other options are in option code sequence)